cnc/server/telemetry_engine.py

```diff
@@ -67,6 +75,12 @@ def handle(self, input_frame):
                     except FileExistsError:
                         logger.error(f"Directory {self.current_path} already exists. Moving on....")
 
+                    # Register drone in redis
+                    # TODO(Aditya): Add drone model to redis
+                    drone_key = f"drone:{extras.drone_id}"
+                    self.r.hset(drone_key, mapping={"last_seen": f"{time.time()}", "status": "idle", "battery": "-1", "model": "anafi"})
```

Comment:
I know we don't have status, battery, and model in the Extras, but soon the new proto will be in place. Why not at least put placeholders there instead of hardcoding the values? i.e mapping={"last_seen": f"{time.time()}", "status": "{extras.status}", "battery": "{extras.battery}", "model": "{extras.model}"})